dozer-types/src/json_types.rs

```diff
@@ -33,13 +80,89 @@ pub fn field_to_json_value(field: Field) -> Value {
         Field::Decimal(n) => Value::String(n.to_string()),
         Field::Timestamp(ts) => Value::String(ts.to_rfc3339_opts(SecondsFormat::Millis, true)),
         Field::Date(n) => Value::String(n.format(DATE_FORMAT).to_string()),
-        Field::Bson(b) => Value::from(b),
+        Field::Json(b) => json_value_to_serde_json(b),
         Field::Point(point) => convert_x_y_to_object(&point.0.x_y()),
         Field::Duration(d) => convert_duration_to_object(&d),
         Field::Null => Value::Null,
     }
 }
 
+pub fn json_value_to_serde_json(value: JsonValue) -> Value {
```

Comment:
We need to get rid of `json_value_to_serde_json` and `serde_json_to_json_value` methods.

All use of `serde_json::Value` can be removed if we properly implement `Serialize` and `Deserialize` for `JsonValue`, but let's leave it in the next PR.

Reply:
Ok!